index.js

```diff
@@ -111,7 +111,20 @@ function launcherStorageServer(config, cb) {
     console.log('not going to start storageServer, shutting down')
     return
   }
-  storageServer = spawn(config.binary_path, [config.ip, config.port])
+  let optionals = []
+  if (config.log_level) {
+    optionals.push('--log-level', config.log_level)
+  }
+  if (config.lokinet_identity) {
```

Comment:
is this the .loki, .snode or ?? address?

Reply:
This is used for the storage server to get the keys used for encrypting the channels with messenger clients, and also is used to calculate the snode address of the server
Open to other ideas, lokid rpc endpoint or something?